src/integrationTest/java/uk/gov/hmcts/darts/usermanagement/controller/UserControllerGetUsersIntTest.java

```diff
@@ -56,11 +59,14 @@ void usersGetShouldReturnForbiddenError() throws Exception {
     }
 
     @Test
-    void usersGetShouldReturnNotImplemented() throws Exception {
+    void usersGetShouldReturnImplemented() throws Exception {
         adminUserStub.givenUserIsAuthorised(mockUserIdentity);
 
-        mockMvc.perform(get(ENDPOINT_URL).queryParam("courthouse", "-1"))
-            .andExpect(status().isNotImplemented());
+        MockHttpServletRequestBuilder requestBuilder = get(ENDPOINT_URL)
+            .header(EMAIL_ADDRESS, "james.smith@hmcts.net")
+            .queryParam(COURTHOUSE_ID, "21");
+
+        mockMvc.perform(requestBuilder).andExpect(status().isOk());
```

Comment:
Please also check the response to assert the user is returned.

Reply:
updated